static/styles/portico/portico_signin.css

```diff
@@ -11,6 +11,11 @@ html {
     min-height: calc(100vh - 402px);
 }
 
+.self-hosted-register-page {
+    /* Footer height (60) + padding of signup form (160) = 220 */
+    min-height: calc(100vh - 220px);
+}
```

Comment:
Would it be easy to make the 60 and 160 into variables, like we have in left-sidebar.css? I feel like that should be our practice rather than writing math like this in a comment, when we can.

Reply:
Done.

Reply:
So, this isn't exactly what I'd had in mind -- the intent was that we'd have a variable that's 30px for the footer height, and use that variable both in the footer CSS and here, so that if we were to make some change to the footer's height, this wouldn't break.

But maybe that's a tricky refactoring, and having names for these is at least more readable than the comments.

What would go wrong if this calculation were to become wrong?

Reply:
Like this 
<img width="1157" alt="image" src="https://user-images.githubusercontent.com/25124304/180150364-832db595-7c71-4a92-aff6-4d62c1643b1e.png">

Reply:
OK, that seems pretty obvious at least.